### DISCUSSION TITLE
Find Type in object graph

### DISCUSSION THREAD

**Discussion by lukematthews** [typescript-eslint/typescript-eslint#9944] [Community Feedback]
Hi Everyone, I'm new here so apologies if it's not the best place for the question.

I'm trying to Lint whether or not an object and all of it properties recursively down the graph contain a particular type.

I started writing a custom rule but finding it difficult to traverse the tree.

Any advice would be appreciated and I'm happy to pass along what we have so far.

Thanks.

**Comment by bradzacher** [comment]
as per our contributing guide for discussions -- we don't use discussions for Q&A
https://typescript-eslint.io/contributing/discussions

Instead hop over to stackoverflow or our discord -- see links above.
